src/frontend/komponenter/Fagsak/Vedtak/AvsnittSkjema.tsx

```diff
@@ -85,6 +84,33 @@ const AvsnittSkjema: React.FC<IProps> = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [avsnitt]);
 
+    function finnBulletpointsFraListe(
+        underavsnittsliste: UnderavsnittSkjemaData[]
+    ): UnderavsnittSkjemaData[] {
+        const bulletpoints: UnderavsnittSkjemaData[] = [];
+        let erBulletpoints = false;
+
+        for (const underavsnitt of underavsnittsliste) {
+            if (erBulletpoints) {
+                bulletpoints.push(underavsnitt);
+            }
+            if (underavsnitt.brødtekst?.startsWith('*-')) {
```

Comment:
Basert på backend-koden og eksemplene så KAN underavsnitt være både slik som dere har beskrevet her, men også 
`*- brødtekstinnhold direkte -*` 

Må kanskje ta stilling til det her også?

ref: https://github.com/navikt/familie-tilbake/blob/96ab3ea4091d4df1632f52056015a6baf69e9318/src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/felles/pdf/DokprodTilHtml.kt#L21

Reply:
Har dere sett på om koden blir mer oversiktlig med reduce her?

```underavsnittliste.reduce((erBulletpoint, underavsnitt) => {
erBulletpoint = (underavsnitt.brødtekst?.startsWith('*-') && underavsnitt.brødtekst.length > 2) || erBulletpoint
if (erBulletPoint) { 
  bulletpoints.push(underavsnitt)
  } 
if (underavsnitt.brødtekst?.startsWith('*-')) { return true }
else if (underavsnitt.brødtekst?.endsWith('-*')) { return false }
else { return erBulletpoint }
}, [])```

Reply:
Hvis man skriver det på den måten, så blir det dessverre stort mellomrom mellom to punkter. Se bildet under. Vi opprettet egen favro på det: https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=NAV-22754

<img width="744" alt="Bulletpoints brev" src="https://github.com/user-attachments/assets/e250f4df-0730-4f10-9d11-a2ae435041a6">